src/components/project/project-member/project-member.service.ts

```diff
@@ -104,7 +59,8 @@ export class ProjectMemberService {
 
     const id = await generateId();
     const createdAt = DateTime.local();
-    await this.repo.verifyRelationshipEligibility(projectId, userId);
+    this.repo.verifyRelationshipEligibility &&
+      (await this.repo.verifyRelationshipEligibility(projectId, userId));
```

Comment:
the following code should work, `await this.repo.verifyRelationshipEligibility(projectId, userId);` if you remove the optional operator, `?`,  in the repo.